reference/customers.v3.yml

```diff
@@ -171,13 +171,18 @@ paths:
              * `segment_ids`- segments the customer belongs to (Beta)
 
              `include=addresses,storecredit,attributes,formfields,shopper_profile_id,segment_ids`
+          explode: false
           schema:
-            type: string
-            enum:
-              - addresses
-              - storecredit
-              - attributes
-              - formfields
+            type: array
+            items:
+              type: string
+              enum:
+                - addresses
+                - storecredit
+                - attributes
+                - formfields
+                - shopper_profile_id
+                - segment_ids
```

Comment:
These two values were mentioned in the description, but were not included in the enum. I can remove if the description wasn't meant to include them.

Reply:
let's leave them.